src/CMakeLists.txt

```diff
@@ -509,6 +509,7 @@ include_directories(
 	${LUA_INCLUDE_DIR}
 	${GMP_INCLUDE_DIR}
 	${JSON_INCLUDE_DIR}
+	${BIT_INCLUDE_DIR}
```

Comment:
I don't see this or the _LIBRARY var defined anywhere

Reply:
See https://github.com/minetest/minetest/blob/d03adb195626069dca5765a20fdca03aaddc1669/CMakeLists.txt#L227